spin/cmds/meson.py

```diff
@@ -946,7 +964,11 @@ def docs(
 
     make_bat_exists = (Path(doc_dir) / "make.bat").exists()
     make_cmd = "make.bat" if sys.platform == "win32" and make_bat_exists else "make"
-    _run([make_cmd, sphinx_target], cwd=doc_dir, replace=True)
+    cmds = [make_cmd, "-j", str(jobs), sphinx_target]
+    if jobs == "auto":
+        cmds = [make_cmd, "-j", sphinx_target]
+
+    _run(cmds, cwd=doc_dir, replace=True)
```

Comment:
This is also harmless and backwards compatible IMO. Just adds support for parallel build of docs.

Reply:
I'm not sure this is backwards compatible, parallel Sphinx builds are breaking regularly (example: https://github.com/scipy/scipy/issues/18300). From the diff I can't quite tell if the default behavior is switched, but the code looks not quite right to me, since if `jobs == 'auto'` then `-j` gets added twice (in lines 950 and 952).

It's also unrelated to the topic of this PR, so I suggest removing the changes. We need this PR to get merged and in a release, so we can move on with `spin` adoption in SciPy - it's important to keep this focused on the issue at hand. If we need other changes unrelated to passing `meson compile` and `meson install`, please put them in a separate PR (or more, one per topic would be great unless things are completely trivial).

Reply:
Sure. No worries. I added this in the beginning because `dev.py` has parallel building of docs available. I think we should remove it from `dev.py` also.